js/main.js

```diff
@@ -678,65 +678,44 @@ function DragAndDropTable(table,mode,queryKey) {
 	 */
 	function FullSizeImage(els) {
 		if (!els) return;
-		els = (typeof els == "object" || typeof els == "function") && typeof els.length == "number"?els:[els];
-		var hashTrigger = null;
-		var body = document.body;
-   		var imageCanvas = document.getElementById("image-canvas") || document.createElementWithAttributes("div", {"id": "image-canvas"}, body);
-		imageCanvas.setVisible = function(visible) {
-			if (visible)
-				this.classList.remove("js-display-none");
-			else
-				this.classList.add("js-display-none");
-		};
-		var stopTrigger = function() {
-			if (hashTrigger) {
-				window.clearInterval(hashTrigger);
-				var scrollPos = document.getScrollPosition();
-				window.history.back();
-				// Fuer den Fall, dass man bei eingeblendeten Bild gescrollt hat
-				window.scrollTo(scrollPos.left, scrollPos.top);
-			}
-		};
+		const isPopoverSupported = () => HTMLElement.prototype.hasOwnProperty("popover");
```

Comment:
I kindly request that you refrain from using lambdas, as they render the code illegible.